Doc/library/math.rst

```diff
@@ -299,8 +299,10 @@ Number-theoretic and representation functions
 .. function:: trunc(x)
 
    Return the :class:`~numbers.Real` value *x* truncated to an
-   :class:`~numbers.Integral` (usually an integer). Delegates to
-   :meth:`x.__trunc__() <object.__trunc__>`.
+   :class:`~numbers.Integral` (usually an integer). Truncating *x* means
+   removing the digits after the decimal comma, hence rounding toward 0. It is
```

Comment:
Since not all locales use a comma:
```suggestion
   removing the digits after the decimal separator, hence rounding toward 0. It is
```

Reply:
Done

Reply:
Done. 
I didn't realize that the English doc had to be normalized. I admit that if I were to translate it to French (my mother tongue), I clearly would have used "dot". But I guess it's great this way, thanks!

> Not a math expert but the change makes sense to me!

Great! That's the point. That the doc is easy to get even without being a math expert. At least in my opinion.